### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alliology934**
Initial addition to the READ.ME. Please everyone add your GitHub profile link to the "Contributors" section using [Markdown styling](https://www.markdownguide.org/basic-syntax/). 

Please also read through the entire thing and make any corrections, edits, and contributions as you see fit.

Woo!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+
+## Description
+Household Manager Web Application using C# and ASP.NET Core 6 via MVC created with the intent for 5 entry-level programmers to practice building an application from scratch utilizing Agile development and version-control via branching in GitHub.
+
+Household Pro allows individuals that live in a multi-person unit to collect, organize, and assign all chores and assignments involved in keeping up with home organization and cleanliness efficiently and faily.
+
+## Contributors
+- [**Alli**](https://github.com/alliology934)
+- [**Erica**](Add your link here like I have above.)
+- [**Michele**](Add your link here like I have above.)
```
Comment (mmrichmond6) [inline README.md]: (https://github.com/mmrichmond6)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
-# group-Jessica
-Household Manager Web Application using C#
+# Household Pro
+
+## Description
+Household Manager Web Application using C# and ASP.NET Core 6 via MVC created with the intent for 5 entry-level programmers to practice building an application from scratch utilizing Agile development and version-control via branching in GitHub.
+
+Household Pro allows individuals that live in a multi-person unit to collect, organize, and assign all chores and assignments involved in keeping up with home organization and cleanliness efficiently and faily.
```
Comment (mmrichmond6) [inline README.md]: fairly (correct misspelling of faily at the end of the sentence)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,42 @@
...
+  - Syncfusion
+  - Identity
+  - Icon API
+  - Font Awesome
+
+## Instructions
+1. Fork this repository
+2. Clone your forked repository
+3. **TODO:** Stuff about downloading packages for the app to work
+4. **TODO:** Stuff about hooking up their local SQL Database
```
Comment (mmrichmond6) [inline README.md]: 4.  Through the Package Manager type "Update-Database" to initialize and migrate the seeded database with its tables to SQL Express.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,42 @@
...
+5. Run the application to see if it builds and runs correctly
```
Comment (mmrichmond6) [inline README.md:41]: Log-in with either the pre-seeded admin or user to visit their authorized pages.

**Review by mmrichmond6 — CHANGES_REQUESTED** [review]
I added comments to the respective lines of code for review/change.  I "think" this is what I have to do for Alli to make the changes then it will be reviewed once more before merging?

File: README.md (hunk 1 above)
Comment (alliology934) [reply README.md]: Change made.

File: README.md (hunk 2 above)
Comment (alliology934) [reply README.md]: Change made.

File: README.md (hunk 3 above)
Comment (alliology934) [reply README.md]: This is exactly what I needed, thanks! Changes made, and I also added a step before this to "Ensure Microsoft SQL Express is installed on your local machine."  I hope that is true. Won't hurt though at the very least, haha.

File: README.md (hunk 4 above)
Comment (alliology934) [reply README.md:41]: Great addition! Change made.

**Comment by alliology934** [comment]
Made Michele's suggested changes (thanks for the great additions, Michele!) and re-submitted the pull request.

**Review by wilsons6561 — APPROVED** [review]

**Review by ejohnson1124 — APPROVED** [review]

**Review by mmrichmond6 — APPROVED** [review]

**Pull request merged**